docs/OnTrack/Front End Migration/Research & Findings/revise_prosconsofdifferentframeworks.md

```diff
@@ -0,0 +1,108 @@
+
+
+In this issue, we will discuss the current state of testing and what tools are we proposing and why we think it is logical to choose that over others. We will discuss some alternative tools and their pros and cons as well which we considered in our decision-making process.
+
+<br>
+
+# What is the current state of testing?
+
+While there are some existing unit tests for components written in TypeScript, there are no existing unit tests for the CoffeeScript components. Existing tests use Karma test runner integrated with Jasmine. Currently, `"ng test"` command (Runs unit tests in a project) and `npm install` is not working due to dependencies issues, which are getting fixed.  
+
+<br>
+<br>
+
+# Comparing different alternatives

```

Comment:
As per above this would be h2

If you want you could put the below pros and cons into a table.